src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -84,7 +86,16 @@
  *
  *      for (;;){}               // OK
  *      for(;;){}                // violation, space after 'for' is required
- *      }
+ *
+ *      try (InputStream ignored = System.in) {} // OK
+ *      try(InputStream ignored = System.in) {} // violation ''try' is not followed by whitespace'
+ *
+ *      try (InputStream ignored = System.in){} //OK
```

Comment:
Remove `()` block
https://docs.oracle.com/javase/tutorial/essential/exceptions/catch.html

It is not required try-catch.

It might allow you to make example code single line

Reply:
Please check if it correct now?